com/metacube/SCF/Assignment8/ZooOperations.java

```diff
@@ -0,0 +1,149 @@
+package GET2018.com.metacube.SCF.Assignment8;
+
+import java.util.Scanner;
+
+public class ZooOperations {
+
+	void addAnimal(AnimalType animalSpecies)
+    {
+        switch(animalSpecies)
+        {
+        case LION:
+            addLion();
+            break;
+        case CROCODILE:
+            addCrocodile();
+            break;
+        case PEACOCK:
+            addPeacock();
+            break;
+        default:
+        	break;
+        }
+        
+    }
+    
+    Scanner scan = new Scanner(System.in);
+    void addLion()
+    {
+        System.out.println("Enter animal ID: ");
+        int animalID = scan.nextInt();
+        System.out.println("Enter animal age: ");
+        int animalAge = scan.nextInt();
+        System.out.println("Enter animal name: ");
+        String animalName = scan.next();
+        System.out.println("Is native from: ");
+        String nativeFrom = scan.next();
+        
```

Comment:
Check for empty and null strings here. Infact you can create a readInput() method and call it every time you read console input.